lib/src/pages/credits.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/material.dart";
+
+import "package:ramaz/widgets.dart";
+import "package:ramaz/data.dart";
+
+import "drawer.dart";
+
+/// Creates the Credits Page by iterating through each ContributorCard.
+class CreditsPage extends StatelessWidget {
+	/// Builds the Credits page.
+	const CreditsPage();
+
+	@override 
+	Widget build (BuildContext context) => ResponsiveScaffold(
+		drawer: const NavigationDrawer(),
+		appBar: AppBar(title: const Text ("Credits")),
+		bodyBuilder: (_) => ListView(
+			children: [
+				for (final Contributor contributor in Contributor.contributors)
+					ContributorCard(contributor)
+			]
+		)
+	);
+}
+/// A class that creates each individual Contributor's Card.
+/// It creates a Row that holds the Image, and then a Column that
+/// holds the rest of the information.
+class ContributorCard extends StatelessWidget{
+	/// contributor is a variable that must be a Contributor.
```

Comment:
```suggestion
	/// The contributor this widget represents.
```

You don't need to include types, since the docs will have a link to `Contributor` automatically.